src/js/actions/layers.js

```diff
@@ -381,6 +381,14 @@ define(function (require, exports) {
         if (typeof layerSpec === "number") {
             layerSpec = [layerSpec];
         }
+        
+        layerSpec.forEach(function (layerID) {
+            var layer = document.layers.byID(layerID);
+
+            if (layer) {
+                throw new Error("Trying to add a layer that is already existed: " + layerID + " " + layer.name);
```

Comment:
Grammar nit: this should either read "Trying to add a layer that already exists" or "Trying to add a layer this is already present".